pkg/registry-disk/registry-disk.go

```diff
@@ -111,7 +112,10 @@ func GenerateContainers(vmi *v1.VirtualMachineInstance, podVolumeName string, po
 			volumeMountDir := generateVolumeMountDir(vmi, volume.Name)
 			diskContainerName := fmt.Sprintf("volume%s", volume.Name)
 			diskContainerImage := volume.RegistryDisk.Image
-
+			resources := kubev1.ResourceRequirements{}
+			resources.Limits = make(kubev1.ResourceList)
+			resources.Limits[kubev1.ResourceCPU] = resource.MustParse("50m")
```

Comment:
How was this value decided ?

Reply:
hmm, just needed to start somewhere.
This container only mounts a path. - I'll make it a global default variable